### ISSUE TITLE
This graph for me is wrong

### ISSUE THREAD

**Issue by Gipielle** [CSSEGISandData/COVID-19#441]
<img width="732" alt="Schermata 2020-03-11 alle 09 26 30" src="https://user-images.githubusercontent.com/53569015/76396923-7519b380-637a-11ea-9a2a-e1926191d737.png">

Good morning,
I think this chart is wrong, the cases of contagion are compared (China) with the recovered (Total)

and not China Confimed with China Recovered or Total Confirmed with Total Recovered.

**Comment by nguyenhychan** [comment]
I agree that it is clearer to split the two recovered statistics, or just simply plot the existing cases directly. It is so confusing to see that the number of recovered cases is close to the total case number in China and it is easy to mistake that all cases in China has recovered.
